java-checks/src/test/files/checks/DisallowedClassCheck.java

```diff
@@ -1,3 +1,9 @@
+import unknown;
+import java.lang.Object;
+import java.lang.String; // Noncompliant
+import java.lang.String.*; // Noncompliant
+import static java.lang.String.valueOf; // Noncompliant
+
 class A {
   private String disallowed; // Noncompliant [[sc=11;ec=17]]
   private Integer disallowed2;
```

Comment:
now that you are editing the file, could you please rename this `disallowed2` Integer to something else? because the Integer class is not disallowed for this test, only  ` visitor.disallowedClass = "java.lang.String";`